### ISSUE TITLE
Unicode TNEFFileName Encoding returns wrong

### ISSUE THREAD

**Issue by umutkirgoz** [QualityUnit/TNEFDecoder#5]
Filenames are returning with corrupted encoding.
```
 function getName()
   {
       if ($this->name_is_unicode) {
           return substr(iconv('utf-16', 'utf-8', $this->name), 0, -1);
       }
       return $this->name;
   }
```
https://github.com/QualityUnit/TNEFDecoder/blob/efa3c74b6c287eae713b72635eb853bfac1675fb/src/TNEFDecoder/TNEFFileBase.php#L50:L55

**Sample Data:**
Before Encoding:
`E\x00K\x00_\x003\x00a\x00.\x001\x00p\x00.\x001\x002\x005\x000\x003\x00-\x001\x002\x009\x009\x006\x00.\x00t\x00i\x00f\x00\x00\x00`

After Encoding (Wrong):
`䔀䬀开㌀愀⸀㄀瀀⸀㄀㈀㔀　㌀ⴀ㄀㈀㤀㤀㘀⸀琀椀昀\x00`

Right Encoding:
`EK_3a.1p.12503-12996.tif`

**Comment by martincivan** [comment]
@umutkirgoz 
Thank you for your report. 
Could you please provide an input file where can we simulate the bug?
Or do you already know how to fix this code? What is the original encoding?

**Comment by martincivan** [comment]
Released in 1.1.0
